sale_by_packaging/models/sale_order_line.py

```diff
@@ -86,9 +96,11 @@ def _force_qty_with_package(self):
     def _onchange_product_uom_qty(self):
         self._force_qty_with_package()
         res = super()._onchange_product_uom_qty()
-        if not res:
-            res = self._check_qty_is_pack_multiple()
-        return res
+        is_pack_multiple_warning = self._check_qty_is_pack_multiple()
+        if is_pack_multiple_warning:
+            self.product_packaging_qty = False
+            self.product_packaging = False
+        return res if res else is_pack_multiple_warning
 
     def _check_qty_is_pack_multiple(self):
```

Comment:
IMO this _check_qty_is_pack_multiple is totally useless as the validation is already done by odoo core.
I removed it here https://github.com/OCA/sale-workflow/pull/1231/ :thinking: